src/components/Slider/styledSlider.ts

```diff
@@ -0,0 +1,44 @@
+import { NavLink } from "react-router-dom";
+import styled from "styled-components";
+import { hoverBtnColor } from "../../styledConstantsColors";
+
+export const SliderContainer = styled.div`
+  position: relative;
+  width: 100%;
+`;
+
+export const Slide = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 24px;
+  min-height: 340px;
+  padding-bottom: 80px;
+`;
+
+export const StyledSliderLink = styled(NavLink)`
+position: absolute;
+right: 0;
+bottom: 15px;
+z-index: 3;
+  padding: 10px 31px;
+  font-size: 20px;
+font-style: normal;
```

Comment:
pretty sure you can position this Link without absolute position, fe using "margin-left: auto". I did my version of this link in my commit.